api/src/routes/user.test.ts

```diff
@@ -369,9 +369,9 @@ describe('userRoutes', () => {
         const setCookie = res.headers['set-cookie'];
         expect(setCookie).toEqual(
           expect.arrayContaining([
-            'jwt_access_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT',
-            '_csrf=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT',
-            'csrf_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT'
+            'jwt_access_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly; Secure; SameSite=Lax',
+            '_csrf=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly; Secure; SameSite=Lax',
+            'csrf_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly; Secure; SameSite=Lax'
```

Comment:
Our current client cannot handle `httpOnly` `csrf_token`s:
https://github.com/ojeytonwilliams/freecodecamp/blob/ad0f272359a66440f7a1a6ca41c0b56c1b52b139/client/src/utils/ajax.ts#L26-L31

Also, how is the `csrf_token` `same_site=lax`? I thought the config made it `strict`?

Reply:
Good catch. What's happening here is that we're the cookies are being 'deleted' by having their values set to nothing and their expiry date set to the past. The point is that the server cannot delete cookies, but this is the standard way to tell the client that it should.

The other attributes come along for the ride because that's how all cookies are set unless we specify otherwise.

AFAIK, there's no downside to this, but I was too lazy to write a test that checked just the bit that matters i.e. `csrf_token=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; `

Since it's very far from obvious what's going on, I'll create a better test.